src/ReachSets/ContinuousPost/GLGM06/reach.jl

```diff
@@ -0,0 +1,76 @@
+# ===============================================================
+# Homogeneous case
+# ===============================================================
+function reach_homog!(HR::Vector{ReachSet{Zonotope, Float64}},
+                      Ω0::Zonotope,
+                      Φ::AbstractMatrix,
+                      N::Int,
+                      δ::Float64,
+                      max_order::Int)
+
+    # save timestamps with the reach set
+    t0, t1 = zero(δ), δ
+
+    # initial reach set
+    HR[1] = ReachSet{Zonotope, Float64}(Ω0, t0, t1)
+
+    k = 1
```

Comment:
Starting with `k = 2` here, you can change the condition to `k <= n` and the access to `HR[k]`.